scripts/qemu_launcher

```diff
@@ -153,7 +163,7 @@ else
     then
         start_qemu "-net nic"
     else
-        cmdline="$cmdline ip=192.168.7.2::192.168.7.1:255.255.255.0"
-        start_qemu "-device virtio-net-pci,netdev=net0,mac=52:54:00:12:34:02 -netdev tap,id=net0,ifname=tap0,script=no,downscript=no"
+        cmdline="$cmdline ip=$target_ip::$server_ip:255.255.255.0"
```

Comment:
Actually, the netmask should be configurable as well, shouldn't it? I guess it depends on how one sets up the tap device though, so maybe not? @BassemMohsen what do you think?

Reply:
the script runqemu has hardcoded the subnet mask, you can look at line 946 here:
https://git.yoctoproject.org/cgit.cgi/poky/plain/scripts/runqemu

I guess it's not the optional for yocto ssh and ping tests.

Reply:
So the static IP and gateway should be better like 192.168.7.x maybe?

Reply:
Also it's a long shot the user would be interested to use subnet masks here. and usually 24 mask is what you use with ip4.

Reply:
If the netmask is hardcoded upstream anyway then nevermind :)